BUILD.bazel

```
@@ -98,3 +99,54 @@ test_suite(
     tags = ["manual"],
     tests = ["//rs/tests/testing_verification/testnets:single_large_node"],
 )
+
+### Pocket IC
+
+# The pocket-ic server binary. Use this as a test dependency if the test
+# does not require a specific pocket-ic version (see ":pocket-ic-variant"
+# for details).
+# By default returns the pocket-ic server from the source tree to ensure
+# consistency within the source tree. See 'pocket_ic_mainnet_test' for
+# overrides.
+alias(
+    name = "pocket-ic-server",
+    actual = select({
+        ":pocket-ic-variant-mainnet": "//:pocket-ic-mainnet",
+        ":pocket-ic-variant-head": "//rs/pocket_ic_server:pocket-ic-server",
+        "//conditions:default": "//rs/pocket_ic_server:pocket-ic-server",
+    }),
+)
+
+# A setting to switch between different variants of pocket-ic. The
+# default pocket-ic variant/version (head) is the one as in the
+# source tree.
+string_setting(
+    name = "pocket-ic-variant",
```

How about being explicit about which pocket-ic "thing" we're mentioning here? pocket-ic usually refers to the library while the following is about the server:
```suggestion
    name = "pocket-ic-server-variant",
```
Same comment for the other things below.

---

yep that makes sense!